onmt/inputters/text_utils.py

```diff
@@ -282,11 +284,13 @@ def _addcopykeys(vocabs, example):
     example['src_ex_vocab'] = src_ex_vocab
 
     if example['tgt'] is not None:
+        tgt = None
         if vocabs['data_task'] == ModelTask.SEQ2SEQ:
             tgt = [DefaultTokens.UNK] + example['tgt']['tgt'].split() \
                   + [DefaultTokens.UNK]
         elif vocabs['data_task'] == ModelTask.LANGUAGE_MODEL:
             tgt = example['tgt']['tgt'].split() \
                   + [DefaultTokens.UNK]
-        example['alignment'] = src_ex_vocab(tgt)
+        if tgt:
```

Comment:
why do we need this change in this section? specific case where it does not work ?